src/app/pages/account/users/user-form/user-form.component.html

```diff
@@ -36,17 +36,17 @@
         <ix-input
           formControlName="password"
           type="password"
-          [label]="'Password' | translate"
-          [tooltip]="(isNew ? tooltips.password : tooltips.password_edit) | translate"
-          [required]="isNew"
+          [label]="((isNewUser ? 'Password' : 'New Password') | translate)"
```

Comment:
This won't be picked up by message extractor. Needs to be `condition ? ('Password' | translate) : ('New Password' | translate)`